include/linux/ancillary_bus.h

```diff
@@ -0,0 +1,62 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+/*
+ * Lightweight software bus
```

Comment:
Please remove this Lightweight word.

it is very confusing and it doesn't serve any meaningful purpose.

actually remove this whole line. Its documentation is already there...

Reply:
removed